src/ExpressionEvaluator/Package/source.extension.vsixmanifest

```diff
@@ -4,13 +4,24 @@
     <Identity Id="21BAC26D-2935-4D0D-A282-AD647E2592B5" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Expression Evaluators</DisplayName>
     <Description xml:space="preserve">Roslyn Expression Evaluators</Description>
```

Comment:
This is fine for now. But will we need to remove the 'Roslyn' moniker from all these VSIX manifests at some point?